app/views/support/multiple_locations/new.html.erb

```diff
@@ -0,0 +1,86 @@
+<%= render PageTitle.new(title: t("support.providers.multiple_locations.new")) %>
+
+    <% content_for :before_content do %>
+      <%= govuk_back_link_to(support_recruitment_cycle_provider_locations_path) %>
+    <% end %>
+
+    <%= form_with(model: @multiple_locations_form,
+                  url: support_recruitment_cycle_provider_multiple_locations_path,
+                  local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <span class="govuk-caption-l"><%= @provider.provider_name %></span>
+
+      <h1 class="govuk-heading-l"><%= t("support.providers.multiple_locations.new") %></h1>
+      <div class="govuk-form-group">
+        <%= f.govuk_text_area(
+              :location_details,
+              hint: {
+                text: "Enter the location name, unique reference number and address. Separate each
+                location item with a tab or comma and use a new line for each location. You can also copy and
+                paste directly from a spreadsheet as long as it is formatted correctly."
+              },
+              label: { text: "Location details", class: "govuk-label--s" }
+            ) %>
+        <%= f.govuk_submit %>
+    <% end %>
+
+    <p class="govuk-body">
+      <%= govuk_link_to(t("cancel"), support_recruitment_cycle_provider_locations_path) %>
+    </p>
+    </div>
+
+    <h2 class="govuk-heading-m">How to format the data</h2>
+
+    <p class="govuk-body">
+      Enter the location name, unique reference number and address. The order is important. You need to include:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>location name</li>
+      <li>unique reference number</li>
+      <li>address line 1</li>
+      <li>address line 2</li>
+      <li>town or city</li>
+      <li>county</li>
+      <li>postcode</li>
+    </ul>
+
+    <h3 class="govuk-heading-s">Copying from a <abbr title="comma separated values">CSV</abbr></h3>
+
+    <p class="govuk-body">
+      Separate each location item with a comma and use a new line for each location. For example:
+    </p>
+
+    <pre class="preformatted-text"><code class="app-code--style">
```

Comment:
Can we sort out the indentation of this block here?

Reply:
The pre tag preserves both spaces and line breaks. The text will be displayed exactly as written in the HTML source code. So the code text will be indented on the page.

Reply:
<img width="1524" alt="image" src="https://user-images.githubusercontent.com/50492247/223469932-7a40a288-9a17-4ac3-abb4-7a6a6a913be1.png">
It would look like this.